config/routes.rb

```diff
@@ -26,6 +26,6 @@
   as :user do
     delete '/sign_out' => 'devise/sessions#destroy', :as => :destroy_user_session
   end
-  get '/login' => 'application#login', as: :login
+  get '/login' => 'application#login', as: :new_session
```

Comment:
this change will probably require changes in other places like in alfred omniauth strategy, did you check this?

Reply:
yes I did